<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sell</class>
 <widget class="QDialog" name="sell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#sell{border-image: url(:/pig/pigPos.jpg);}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:30px
}</string>
     </property>
     <property name="text">
      <string>可以出售的大猪</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;站酷小薇LOGO体&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPrice">
         <property name="styleSheet">
          <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:20px
}</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnSell">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
     </property>
     <property name="text">
      <string>出售</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
